packages/contracts/scripts/deploy/dollar/solidityScripting/01_Diamond.s.sol

```diff
@@ -146,6 +151,28 @@ contract DiamondScript is Constants {
         IAccessControl = AccessControlFacet(address(diamond));
         StakingFacet IStakingFacet = StakingFacet(address(diamond));
         IStakingFacet.setBlockCountInAWeek(420);
+
+        // init UbiquityPool
+        // add collateral LUSD token
+        uint256 poolCeiling = 50_000e18; // max 50_000 of collateral tokens is allowed
```

Comment:
Probably should be initialized to at least 500k

Reply:
it's a market cap to resguard minting, but its more like a consensus value

Reply:
https://github.com/ubiquity/ubiquity-dollar/blob/b7f59cf64e97f5366099ea29accff3224e1817b6/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L64

Reply:
> Where do we get this number from? I mean how it was decided to set 50k LUSD as max collateral? (just curious).

The original [FraxPoolV3](https://etherscan.io/address/0x2fE065e6FFEf9ac95ab39E5042744d695F560729) has 50k pool ceiling for the `LUSD` collateral

> Probably should be initialized to at least 500k

The pool ceiling is a safety measure to limit losses in case of a critical vulnerability in the contract. Basically we should start simple and set `LUSD` pool ceiling to some small amount like 10k to see how it goes. Then, with the increase of the incoming liquidity, we should gradually increase the pool ceiling (there is a separate [method](https://github.com/ubiquity/ubiquity-dollar/blob/4f679f887405ec6287ba6b9cab097a3910c8aacb/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L656) for it) up to any amount.

Frax basically pauses pools when pool ceiling for collateral is reached so that AMO minters could borrow collateral to make yield while users are left with `FRAX` tokens tradeable only on second markets.